README.md

```diff
@@ -507,9 +336,15 @@ If you'd like to contribute to this project, you'll first need to clone this rep
 Then, you will need to run the following commands:
 
 1. `poetry install` -- installs the necessary dependencies into a poetry-managed virtual environment.
-2. `poetry run pre-commit install` -- creates a git [pre-commit](https://pre-commit.com) hook which will automatically run formatters, style checks, etc. against your proposed commits.
+2. ~`poetry run pre-commit install` -- creates a git [pre-commit](https://pre-commit.com) hook which will automatically run formatters, style checks, etc. against your proposed commits.~
```

Comment:
This should be removed rather than struck out

Reply:
Agree -- however, I *hope* to bring it back very soon with a full run of the formatter(s) over the repo...

Reply:
Changed tack and removed mention of the pre-commit hook until I actually run the formatter.